arch/arm/boot/dts/imx6ul-ts7250v3.dtsi

```diff
@@ -0,0 +1,1030 @@
+			compatible = "technologic,supervisor-adc";
+			#io-channel-cells = <1>;
+		};
+
+		supervisor_temp: supervisor_temp {
+			compatible = "technologic,supervisor-temp";
+			#io-channel-cells = <0>;
+		};
+	};
+
+	m41t00s: rtc@68 {
+		compatible = "m41t00";
+		reg = <0x68>;
+		status = "disabled";
+	};
+
+	supervisor_rtc: rtc@68 {
+		compatible = "technologic,supervisor-rtc";
+		reg = <0x68>;
+		wakeup-source;
+		status = "disabled";
+	};
+
+	ism330: gyro@6a {
+		compatible = "st,ism330dlc";
+		reg = <0x6a>;
+		interrupt-parent = <&gpio5>;
+		interrupts = <0 IRQ_TYPE_LEVEL_HIGH>;
+	};
+};
+
+&adc1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_adc1>;
+	vref-supply = <&vref_adc_2v5>;
+	status = "okay";
+};
+
+&can1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_flexcan1>;
+	status = "okay";
+};
+
+&can2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_flexcan2>;
+	status = "okay";
+};
+
+&ecspi3 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_ecspi3>;
+	num-cs = <1>;
+	cs-gpios = <&gpio4 12 GPIO_ACTIVE_LOW>;
+	status = "okay";
+
+	wilc3000: wifi@0 {
+		compatible = "microchip,wilc3000";
+		reg = <0>;
+		spi-max-frequency = <20000000>;
+		reset-gpios = <&gpio5 5 GPIO_ACTIVE_HIGH>;
+		chip_en-gpios = <&gpio5 6 GPIO_ACTIVE_HIGH>;
+		interrupt-parent = <&gpio3>;
+		interrupts = <1 IRQ_TYPE_EDGE_FALLING>;
+	};
+};
+
+&fec1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_enet1>;
+	phy-mode = "rmii";
+	phy-handle = <&ethphy0>;
+	phy-supply = <&reg_3v3>;
+
+	status = "okay";
+	mdio {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		ethphy0: ethernet-phy@0 {
+			clocks = <&clks IMX6UL_CLK_ENET_REF>;
+			clock-names = "rmii-ref";
+			compatible = "ethernet-phy-ieee802.3-c22";
+			reg = <2>;
+		};
+
+		ethphy1: ethernet-phy@1 {
+			clocks = <&clks IMX6UL_CLK_ENET_REF>;
+			clock-names = "rmii-ref";
+			compatible = "ethernet-phy-ieee802.3-c22";
+			reg = <1>;
+		};
+	};
+};
+
+&fec2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_enet2>;
+	phy-mode = "rmii";
+	phy-handle = <&ethphy1>;
+	phy-supply = <&reg_3v3>;
+	status = "okay";
+};
+
+&gpio1 {
+	gpio-line-names =
+		"AN_CH1", "AN_CH2", "I2C_1_CLK", "I2C_1_DAT", "EN_SD_CARD_3.3V",
+		"AN_CH3", "ETH_MDIO", "ETH_MDC", "AN_CH4", "AN_CH5", "",
+		"SEL_XBEE_USB", "", "FPGA_RESET", "MAGNET_IRQ", "",
+		"CONSOLE_TXD", "CONSOLE_RXD", "EN_RED_LED#", "EN_GRN_LED#",
+		"", "EN_XBEE_USB", "CAN_2_TXD", "CAN2_RXD_3V", "UART3_TXD",
+		"UART3_RXD", "UART3_CTS#", "UART3_RTS#", "UART4_TXD",
+		"UART4_RXD", "EN_DIO_FET", "NIM_PWR_ON";
+};
+
+&gpio2 {
+	gpio-line-names =
+		"ENET1_RX_DATA0", "ENET1_RX_DATA1", "ENET1_RX_EN",
+		"ENET1_TX_DATA0", "ENET1_TX_DATA1", "ENET1_TX_EN",
+		"ENET1_TX_CLK", "ENET1_RX_ER", "ENET2_RX_DATA0",
+		"ENET2_RX_DATA1", "ENET2_RX_EN", "ENET2_TX_DATA0",
+		"ENET2_TX_DATA1", "ENET2_TX_EN", "ENET2_TX_CLK",
+		"ENET2_RX_ER", "EMMC_CMD", "EMMC_CLK", "EMMC_D0", "EMMC_D1",
+		"EMMC_D2", "EMMC_D3", "", "", "", "", "", "", "", "", "", "";
+};
```

Comment:
For future reference, I don't think its necessary, and it actually may be detrimental, to name pins as GPIO which generally cannot be GPIO. e.g. all of the ENET pins only go to the PHY, and while they could be put in to GPIO mode, are not usable as GPIO normally on this system. Saves us time in labeling and reduces noise in `gpioinfo` to end users.